.jenkins/pipelines/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+                sh """
+                   # Initialize dependencies repo
+                   ${JENKINS_SCRIPTS}/global/init-config.sh
+
+                   # Install global dependencies
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/global/init-install.sh
+                   """
+            }
+        }
+        stage('Init Code Coverage Config') {
+            when {
+                expression { params.TEST_CONFIG == 'Code Coverage' }
+            }
+            steps {
+                sh """
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/code-coverage/init-install.sh
+                   """
+            }
+        }
+        stage('Build repo source') {
+            steps {
+                sh """
+                   ${JENKINS_SCRIPTS}/global/make-world.sh
+                   """
+            }
+        }
+        stage('Run all tests') {
+            steps {
+                sh """
+                   ${JENKINS_SCRIPTS}/global/make-tests.sh
+                   """
+            }
+        }
+/*
+        stage('Setup Solutions Access') {
+            steps {
+                withCredentials([string(credentialsId: 'Jenkins-ServicePrincipal-ID', variable: 'SERVICE_PRINCIPAL_ID'),
+                                 string(credentialsId: 'Jenkins-ServicePrincipal-Password', variable: 'SERVICE_PRINCIPAL_PASSWORD'),
+                                 string(credentialsId: 'ACC-Prod-Tenant-ID', variable: 'TENANT_ID'),
+                                 string(credentialsId: 'ACC-Prod-Subscription-ID', variable: 'AZURE_SUBSCRIPTION_ID'),
+                                 string(credentialsId: 'oe-jenkins-dev-rg', variable: 'JENKINS_RESOURCE_GROUP'),
+                                 string(credentialsId: 'mystikos-managed-identity', variable: "MYSTIKOS_MANAGED_ID")]) {
+                    sh '''
+                       ${JENKINS_SCRIPTS}/solutions/init-config.sh
+                       ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                       ${JENKINS_SCRIPTS}/solutions/azure-config.sh
+                       '''
+                }
+            }
+        }
+        stage('Run SQL Solution - USEAST') {
+            steps {
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                    withCredentials([string(credentialsId: 'mystikos-sql-db-name-useast', variable: 'DB_NAME'),
+                                     string(credentialsId: 'mystikos-sql-db-server-name-useast', variable: 'DB_SERVER_NAME'),
+                                     string(credentialsId: 'mystikos-maa-url-useast', variable: 'MAA_URL'),
+                                     string(credentialsId: 'mystikos-managed-identity-objectid', variable: 'DB_USERID')]) {
+                        sh "make tests -C ${WORKSPACE}/solutions"
+                    }
+                }
+            }
+        }
+        stage('Run SQL Solution - CACENTRAL') {
```

Comment:
Are both SQL solution for USEAST and CANADACENTRAL necessary? Nightly tests and CC tests only ran one of CANADACENTRAL or USEAST (respectively). Running both will add an hour to the test.

Reply:
I'll use USEAST for this pipeline